src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Dependencies/AbstractDependencyExplorerCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    /// <summary>
+    ///     Provides the <see langword="abstract"/> base class for commands
+    ///     that handle Explorer-like commands for dependency <see cref="IProjectTree"/>
+    ///     nodes.
+    /// </summary>
+    internal abstract class AbstractDependencyExplorerCommand : AbstractProjectCommand
```

Comment:
This and its implementations are specific to dependency tree items, so should be moved to `Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Commands`.